core/src/main/java/org/apache/calcite/rel/metadata/RelMdMaxRowCount.java

```diff
@@ -159,7 +159,7 @@ && allGroupKeysAreConstant(rel, predicateList)) {
     return rowCount * rel.getGroupSets().size();
   }
 
-  private static boolean allGroupKeysAreConstant(Aggregate aggregate,
+  static boolean allGroupKeysAreConstant(Aggregate aggregate,
```

Comment:
protected or public might be better?

Reply:
We want to reuse this in RelMdRowCount, so I think it's better to have it package-private.